README.md

```diff
@@ -32,3 +44,8 @@ pub fn main() {
   promise.resolve(Ok(Nil))
 }
 ```
+
+Documentation can be found at [https://hexdocs.pm/gleam_fetch](https://hexdocs.pm/gleam_fetch).
+
+`gleam_fetch` works on every JavaScript runtime implementing `fetch`, which
+implies all modern browsers, Node.js >= 18.0.0, Deno & Bun.
```

Comment:
```suggestion
```

Fetch has been around for a long time so we don't need to warn people about support for it